src/clj_sqs_extended/core.clj

```diff
@@ -72,43 +73,58 @@
   (.deleteQueue sqs-client url))
 
 (defn send-message
-  [sqs-client url message]
-  (.sendMessage sqs-client url message))
+  ([sqs-client url body]
+   (send-message sqs-client url body {}))
+  ([sqs-client url body
+    {:keys [format]
+     :or   {format :transit}
+     :as   opts}]
+   (let [payload (serdes/serialize body format)]
```

Comment:
same here, `payload` is only used once, so just thread `->>` this through and save a `let`. I get that sometimes it's useful for readability. In this case though, `body` is obvious enough already.